package.json

```diff
@@ -15,19 +15,19 @@
     "build": "kolibri-tools build prod --file ./build_tools/build_plugins.txt && yarn run hashi-build",
     "makemessages": "kolibri-tools build i18n --file ./build_tools/build_plugins.txt",
     "watch": "kolibri-tools build dev --file ./build_tools/build_plugins.txt",
-    "watch-hot": "kolibri-tools build dev --file ./build_tools/build_plugins.txt --hot",
+    "watch-hot": "yarn run watch --hot",
     "django-devserver": "kolibri --debug manage runserver --settings=kolibri.deployment.default.settings.dev \"0.0.0.0:8000\" ",
-    "devserver": "npm-run-all --parallel django-devserver watch lint:watch hashi-dev",
-    "devserver-hot": "npm-run-all --parallel django-devserver watch-hot lint:watch",
+    "devserver": "npm-run-all --parallel django-devserver fmt-frontend:watch fmt-backend:watch hashi-dev watch",
+    "devserver-hot": "npm-run-all --parallel django-devserver fmt-frontend:watch fmt-backend:watch hashi-dev watch-hot",
     "bundle-stats": "kolibri-tools build stats --file ./build_tools/build_plugins.txt",
     "clean": "kolibri-tools build clean --file ./build_tools/build_plugins.txt",
     "preinstall": "node ./packages/kolibri-tools/lib/npm_deprecation_warning.js",
-    "lint": "kolibri-tools lint",
-    "lint-all": "yarn run lint '{kolibri*/**/assets,packages}/**/*.{js,vue,scss,less,css}' -i '**/node_modules/**','**/static/**','**/packages/kolibri-core-for-export/**'",
-    "lint-all:fix": "yarn run lint-all -w",
-    "lint:fix": "yarn run lint -w",
-    "lint:watch": "yarn lint-all -w -m",
-    "generate-locale-data": "node ./build_tools/i18n/intl_code_gen.js",
+    "fmt-frontend": "yarn run fmt-frontend:check --write",
+    "fmt-frontend:check": "kolibri-tools lint '{kolibri*/**/assets,packages}/**/*.{js,vue,scss,less,css}' --ignore '**/node_modules/**','**/static/**','**/packages/kolibri-core-for-export/**'",
```

Comment:
Need to update the tox.ini frontend linting job for this (this is why tests are failing currently).